datarepo-clienttests/src/main/java/utils/KubernetesClientUtils.java

```diff
@@ -258,7 +259,7 @@ public static void deleteRandomPod() throws ApiException, IOException {
                 pod ->
                     deploymentComponentLabel.equals(
                         pod.getMetadata().getLabels().get(componentLabel)))
-            .skip((int) (podCount * Math.random()))
+            .skip(new Random().nextInt((int) podCount))
```

Comment:
Thanks for fixing this bug!

Reply:
Well, not really a bug -- your version is easier to read in my opinion. But spotbugs was upset.